src/SyncCommand.php

```diff
@@ -45,6 +45,13 @@ public function execute(InputInterface $input, OutputInterface $output): int
         $projectVersionMap = $data->getProjectVersionMap();
         $logger->debug("Retrieved projects: {projects}", ['projects' => print_r($projectVersionMap, true)]);
 
+        // Log projects with no known version.
+        $unknownProjects = array_filter($projectVersionMap, 'is_null');
+        $logger->notice('Projects with no known version: {projects}', ['projects' => print_r($unknownProjects, true)]);
```

Comment:
```suggestion
        $logger->notice('Projects with no known version: {projects}', ['projects' => print_r(array_keys($unknownProjects), true)]);
```

Some minor cleanup of the output.

Reply:
Done.